rg_instructor_analytics_log_collector/processors/base_pipeline.py

```diff
@@ -32,6 +32,19 @@ class BasePipeline(object):
     """
     processor_name = None
 
+    def is_valid(self, result):
```

Comment:
Kinda looks unnecessary, it just proxies the result of another method and does not add anything to it.

Reply:
Reverted the changes

Reply:
Although, I was planning to do more with it e.g. checking required fields, either predefined somewhere in configs or taken from models whenever possible as suggested; but it turned to be much faster to revert the changes and leave validators as is :)